src/components/OrderBookWidget.tsx

```diff
@@ -67,9 +69,9 @@ interface ProcessedItem {
   price: number
 }
 
-const orderbookUrl = (baseToken: TokenDetails, quoteToken: TokenDetails, networkId?: number): string => {
+const orderbookUrl = (baseToken: TokenDetails, quoteToken: TokenDetails, hops: number, networkId?: number): string => {
   const network = getNetworkFromId(networkId || 1)
-  return `https://price-estimate-${network}.dev.gnosisdev.com/api/v1/markets/${baseToken.id}-${quoteToken.id}?atoms=true`
+  return `https://price-estimate-${network}.dev.gnosisdev.com/api/v1/markets/${baseToken.id}-${quoteToken.id}?atoms=true&hops=${hops}`
```

Comment:
Should we be using the prod endpoints here?

Reply:
I'll do it in a different PR then